tests/apps/search/test_partial_mappings.py

```diff
@@ -162,3 +162,51 @@ def test_partial_mappings_french_diacritics(self):
             )["tokens"][0]["token"],
             "electricit",
         )
+
+    def test_partial_mappings_code(self):
+        """Make sure our code analyzer works as expected."""
+        index_name = "stub_index"
+
+        # Create the index and set a mapping that includes the pattern we want to test
+        self.indices_client.create(index=index_name)
+        # The index needs to be closed before we set an analyzer
+        self.indices_client.close(index=index_name)
+        self.indices_client.put_settings(body=ANALYSIS_SETTINGS, index=index_name)
+        self.indices_client.open(index=index_name)
+
+        self.assertEqual(
+            list(
```

Comment:
Why are you enforcing a list() on a list you've just constructed inside ?

Reply:
actually it's a list comprehension, so technically a generator so I need to make it a list to compare it. There may be other ways to compare list?

Reply:
@sveetch apparently it works without converting to a list :man_shrugging: